docs/taskruns.md

```diff
@@ -447,6 +449,43 @@ spec:
   status: "TaskRunCancelled"
 ```
 
+
+### Debugging a `TaskRun`
```

Comment:
Again probably worth mentioning that this is an alpha feature, with a link to the "Alpha Features" section of `docs/install.md`.